requirements.txt

```diff
@@ -23,6 +23,7 @@ pika == 1.2.0
 git+https://github.com/metabrainz/brainzutils-python.git@v2.3.1
 spotipy == 2.17.1
 git+https://github.com/metabrainz/data-set-hoster.git@v-2021-02-10.0#egg=datasethoster
+git+https://github.com/metabrainz/troi-recommendation-playground.git#egg=troi
```

Comment:
Let's still pin it to some commit. lest we run the risk of breaking LB when working on troi.

Reply:
Done.

Reply:
Thanks!